README.md

```diff
@@ -107,6 +123,12 @@ This can be useful when you mutate data without changing the `totalItems`. Also
 
 _We are discussing the naming of this API, as well as whether it should exist at all, in [#26](https://github.com/valdrinkoshi/virtual-scroller/issues/26). The aforementioned [#29](https://github.com/valdrinkoshi/virtual-scroller/issues/29) is also relevant._
 
+### `scrollTo(index: number, offset: number = 0)` method

```

Comment:
rename to scrollItemIntoView w/ similar signature of Element.scrollIntoView? Maybe on the item.scrollIntoView?